go/flags/endtoend/vtctldclient.txt

```diff
@@ -104,52 +105,63 @@ Available Commands:
   VDiff                       Perform commands related to diffing tables involved in a VReplication workflow between the source and target.
   Validate                    Validates that all nodes reachable from the global replication graph, as well as all tablets in discoverable cells, are consistent.
   ValidateKeyspace            Validates that all nodes reachable from the specified keyspace are consistent.
-  ValidateSchemaKeyspace      Validates that the schema on the primary tablet for shard 0 matches the schema on all other tablets in the keyspace.
+  ValidatePermissionsKeyspace Validates that the permissions on the primary of the first shard match those of all of the other tablets in the keyspace.
+  ValidatePermissionsShard    Validates that the permissions on the primary match all of the replicas.
+  ValidateSchemaKeyspace      Validates that the schema on the primary tablet for the first shard matches the schema on all other tablets in the keyspace.
+  ValidateSchemaShard         Validates that the schema on the primary tablet for the specified shard matches the schema on all other tablets in that shard.
   ValidateShard               Validates that all nodes reachable from the specified shard are consistent.
-  ValidateVersionKeyspace     Validates that the version on the primary tablet of shard 0 matches all of the other tablets in the keyspace.
+  ValidateVersionKeyspace     Validates that the version on the primary tablet of the first shard matches all of the other tablets in the keyspace.
   ValidateVersionShard        Validates that the version on the primary matches all of the replicas.
   Workflow                    Administer VReplication workflows (Reshard, MoveTables, etc) in the given keyspace.
+  WriteTopologyPath           Copies a local file to the topology server at the given path.
   completion                  Generate the autocompletion script for the specified shell
   help                        Help about any command
 
 Flags:
-      --action_timeout duration                timeout to use for the command (default 1h0m0s)
```

Comment:
What changes in these flags? I can't tell from the diff.

Reply:
The tmclient related flags were added because of go/cmd/vtctldclient/plugin_grpctmclient.go. Which was needed for vtctldclient to be a full replacement for vtctl.